README.md

```diff
@@ -39,17 +39,18 @@ All contributors are expected to uphold our [Code of Conduct](.github/CODE_OF_CO
 ## Usage
 
 ```python
-from src.oaklib.resource import OntologyResource
-from src.oaklib.implementations.sqldb.sql_implementation import SqlImplementation
-
-resource = OntologyResource(slug='tests/input/go-nucleus.db', local=True)
-oi = SqlImplementation(resource)
-for curie in oi.basic_search("cell"):
-    print(f'{curie} ! {oi.label(curie)}')
-    for rel, fillers in oi.outgoing_relationship_map(curie).items():
-        print(f'  RELATION: {rel} ! {oi.label(rel)}')
+from oaklib import OntologyResource
+
+ontology_resource = OntologyResource(slug='tests/input/go-nucleus.db', local=True)
+ontology_interface = ontology_resource.materialize("sql")
+# can also pass an implementation class explicitly instead of a string.
+
+for curie in ontology_interface.basic_search("cell"):
+    print(f'{curie} ! {ontology_interface.get_label_by_curie(curie)}')
+    for rel, fillers in ontology_interface.get_outgoing_relationship_map_by_curie(curie).items():
+        print(f'  RELATION: {rel} ! {ontology_interface.get_label_by_curie(rel)}')
         for filler in fillers:
-            print(f'     * {filler} ! {oi.label(filler)}')
+            print(f'     * {filler} ! {ontology_interface.get_label_by_curie(filler)}')
```

Comment:
I think `get_label_by_curie` has been deprecated in favor of the `label` method

Reply:
Addressed! Let's merge this.